react-app/src/modules/account/Account.js

```diff
@@ -0,0 +1,120 @@
+import scrypt from 'scrypt-async';
+import nacl from 'tweetnacl';
+import sha256 from 'fast-sha256';
+import blake from 'blakejs';
+
+nacl.util = require('tweetnacl-util');
+
+const OFFSET_MIN = -12;
+const OFFSET_MAX = 14;
+
+class Account {
+  constructor() {
+    this._initialized = false;
+  }
+
+  initialize(username, password, offset) {
+    if (this._initialized) throw new ErrorAccountAlreadyInitialized();
+
+    this.offset = offset;
```

Comment:
Can't we get the offset and tz from Date() or something from js? Also, you're not setting _tz anywhere, or am I missing something?

Reply:
I thought it will be better to set the offset by using a parameter instead of set its value by default to the current timezone. I think it will be a little more flexible implementation.

```this.offset = offset``` calls the offset setter where ```_offset``` and ```_tz``` are set https://github.com/fran-costa/beautifulthings/blob/2a0ca76fb0e76bb998e7835e49aacfb0c2343413/react-app/src/modules/account/Account.js#L96-L103

Reply:
JS has a getTimezoneOffset() method (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset) that returns the timezone offset in minutes, which is great to deal with 30' and 45' offsets. We could (and I think that we should) implement it as is. Is there any problem with this in the server-side? I found this after doing the PR, my mistake.